HW02_B_Mimic_starter.Rmd

```diff
@@ -41,8 +41,12 @@ data("diamonds")
 
 Using the diamonds dataset, make this graph:
 ```{r graph1 code, echo=FALSE}
-
-
+diamonds %>% 
+  ggplot(aes(x = cut, fill = clarity)) +
+  geom_bar(stat = "count", position = "dodge") + # stat argument allows for the y axis to count total number of cases #default of position is #stack, which is not what we want. Dodge will create this staggered looking bar plot
+  labs(title = "My Diamond Collection", subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond", x = "Diamond Cut", y = "Number of Diamonds") + # using " \n " causes a line break in the subtitle
+  theme(plot.title = element_text(hjust = 0.5, size = 20), plot.subtitle = element_text(size = 16))
+#I am struggling to add the rectangle and text over the "Ideal" bars - any advice?

```

Comment:
I am not sure if this correct, but this is what I ended up doing: 
I create two annotation layers: one for the rectangle and a second for the text. This is my code: 
 annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, fill = "black", alpha = 0.2) +
 annotate("text", x = 4, y = 4500, label = "My Best Diamonds,\nof course")
I assumed it was a transparent black rectangle because you can see the gridlines lines and bars. I positioned the text according to the gridlines of the plot. Unlike the legend.position() function, the x and y coordinates are not bound by 0 and 1.